src/pages/login/Login.js

```diff
@@ -1,21 +1,40 @@
-import React from 'react';
-import { Link } from 'react-router-dom'
+import { useAuth } from 'context/authContext';
+import React, { useState } from 'react';
+import { Link, useLocation, useNavigate } from 'react-router-dom';
 // css 
 import './Login.css'
 
 function Login() {
+    const { loginUser } = useAuth();
+    const [userName, setUserName] = useState('')
+    const [password, setPassword] = useState('')
+    const location = useLocation()
+    const navigate = useNavigate()
+    const updateUserName = event => {
+        setUserName(event.target.value)
+    }
+    const updatePassword = event => {
+        setPassword(event.target.value)
+    }
+    const submitLogin = (e) => {
+        e.preventDefault()
+        if (userName !== '' && password !== '') {
+            loginUser()
+            navigate(location?.state?.from?.pathname)
```

Comment:
here use {replace:true} seprated by comma so that when navigating back through back button login doesn't show up again.

Reply:
thanks for your suggestion, updated the code